Automattic/wp-calypso
# FSE: Remove notice for block patterns new location.

#### Changes proposed in this Pull Request

This PR essentially reverts #41963, which introduced the block patterns sidebar to point people to their new location. That sidebar was always meant to be temporary -- it's been in for about 2 weeks, so we're going to remove it in the next FSE plugin version.

#### Testing instructions

* Build / Sync FSE plugin on this branch.
* Go to the editor, ensure the block patterns Icon is no longer in the top-right toolbar and in the dropdown menu.
* Verify that everything else still works as before, and that there are no errors (also check the browser console).

<hr/>

![Screen Shot 2020-05-07 at 7 36 19 PM](https://user-images.githubusercontent.com/28742426/81354813-8a6d3e80-909a-11ea-9fd3-8ebcf682624c.png)
<hr/>

![Screen Shot 2020-05-07 at 7 36 37 PM](https://user-images.githubusercontent.com/28742426/81354811-89d4a800-909a-11ea-8531-3c6381880086.png)
<hr/>
